dep/conda/Makefile

```diff
@@ -1,8 +1,8 @@
 # Build conda environments
 
 # Define
-MINICONDA_INSTALLER = Miniconda3-4.5.11-Linux-x86_64.sh
-
+MINICONDA_INSTALLER = Miniconda3-4.7.12-Linux-x86_64.sh
```

Comment:
I encountered the same error as mentioned in [this](https://stackoverflow.com/questions/56084960/condavalueerror-malformed-version-string-invalid-characters) Stackoverflow ticket.